### ISSUE TITLE
[Design system V2] Content Tabs

### ISSUE THREAD

**Issue by nloureiro** [ethereum/ethereum-org-website#9440]
## Intro
This issue is a part of the [Design System V2 Epic](https://github.com/ethereum/ethereum-org-website/issues/9431) and can be assigned to anyone who wants to collaborate. 
You can use the comments to sign that you want to work on this, and this issue can be assigned to you.
If you have any questions about it, please don't hesitate to reach out on the comments or on our discord

<br>
<br>

## Description
There are different methods of displaying tabular content in production, and this aims to unify how to use tabs in different situations for a more consistent user experience.

 [[Ethereum.org](http://ethereum.org/)](http://ethereum.org/) tabs can range from "simple/classic tabs" to “stablecoins” page tabs that feature more diverse content. 

Tabs are a great tool for creating a more consistent user experience, and it is important to understand the different options available in order to make the most out of their use.

For example, tabs can be used to organize large amounts of information in a more organized, intuitive way, or to make it easier for users to navigate.

- Type: unification design
- Difficulty: 8 out of 10

[Link to the Figma file](https://www.figma.com/file/oO5AnDzeJdXoWpj3iP1U8c/V2%2FContent-Tabs?node-id=0%3A1&t=IVlC7BfT4MeVQXgT-1) to be worked on 

<br>
<br>

## workflow
These are the main steps for each task.

1. #### Use a separate [Figma file](https://www.figma.com/file/oO5AnDzeJdXoWpj3iP1U8c/V2%2FContent-Tabs?node-id=0%3A1&t=IVlC7BfT4MeVQXgT-1).
    - made by the team and liked here should be
2. #### Identify on which pages the component is used.
    - This can be made collaboratively and consists of screenshots/links to the pages where these components are already in use. Note that we can propose the unification of similar components in one if we find that the UX/UI justifies that’s the core unification goal of the Design system V2.
3. #### Identify the chakra component to be used as a base (if any)
    - We use chakra UI as a base framework, and if we can match the new design with an existing UI component in chakra, it’s a good call. If not, we can build one, but this should be something to identify now that will ease the implementation.
4. #### Work on the design.
    - This is where we build the new proposal that will be a new component in the design system, never forgetting the different screen sizes, behavior states, and themes.
5. #### Test and validate the new component on Figma.
    - Before closing the design, we need to test some real use cases. I suggested having some production screenshot as the base and building it on a Figma page with real content to test the live application of the new design.
6. #### Validation and feedback.
    - At this point, we are ready to validate and collect feedback. We should act upon this and have the best proposal possible.
7. #### Spec to handover to code.
    - This will be the final step, where we review the file from the perspective of a handover to the design system master file and to the code or storybook for code development.
8. #### Merge with the master Figma file of the Design system.
    - This step will be made by someone from the core team because there are some things that could be improved in how we can onboard external contributors to the Figma team. It’s a Figma thing that we can’t change for now.
9. #### Close the issue and publish the Design system update on the Figma community**.
    - For the last action, we need to close the issue and update the Design system on the Figam community file. At this point, another issue might need to be created to update the storybook and the code to reflect this new design.

<br>
<br>

**Comment by nloureiro** [comment]
For 2024, we will create a new set of PRs for the Design Community to work on, keeping this on does not make sense anymore.

**Comment by ddannehh** [comment]
Picked this one up, will report back soon. ✌️

**Comment by nloureiro** [comment]
oh my!! this is awesome!

Thank you!!! I need to set some time to review and merge it o the main file!!! 

but at a glance, it looks awesome!!

**Comment by ddannehh** [comment]
You're welcome!
They definitely need to be looked at when considered for the main DS file since the disconnect between the two and not being able to use the main files' components/vars/styles in this.

**Comment by nloureiro** [comment]
> You're welcome! They definitely need to be looked at when considered for the main DS file since the disconnect between the two and not being able to use the main files' components/vars/styles in this.

@ddannehh, let me know if there is anything else you want to work on to complete the DS. I'm open to ideas or components that I might have missed or created an issue for.

**Comment by ddannehh** [comment]
> @ddannehh, let me know if there is anything else you want to work on to complete the DS. I'm open to ideas or components that I might have missed or created an issue for.
Coincidently I've just shared some thoughts on another ticket! :D [#9436 ](https://github.com/ethereum/ethereum-org-website/issues/9436#issuecomment-2082765365)

**Comment by ddannehh** [comment]
@nloureiro I've added these in the main DS file and relinked all styles/vars, should be working :)

**Comment by wackerow** [comment]
Hey folks! Circling through; @nloureiro are we waiting on a review from you for any of this? What are next steps here?
